### PR TITLE
Introduce yaml standard library module

### PR DESCRIPTION
**Pull request by madhukaw**

### CODE REVIEW

File: ballerina/build.gradle [hunk 1]
```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (MohamedSabthar) [inline ballerina/build.gradle]:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment (MohamedSabthar) [reply ballerina/build.gradle]: Check other places

Comment (madhukaw) [reply ballerina/build.gradle]: Fixed in [4e47cc4](https://github.com/ballerina-platform/module-ballerina-yaml/pull/3/commits/4e47cc471807c60f8b218633f36db395cca8d51e)

File: docs/spec/spec.md [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
-Ballerina YAML parser provides APIs to convert a YAML configuration file to `json`, and vice-versa. The module supports both the functions of read and write either a single YAML document or a YAML stream.
+_Owners_: @shafreenAnfar @MadhukaHarith92  
+_Reviewers_: @shafreenAnfar  
+_Created_: 2023/04/04  
+_Updated_: 2023/04/04  
+_Edition_: Swan Lake
+
+## Introduction
+This is the specification for the Yaml standard library of [Ballerina language](https://ballerina.io/), which provides APIs to convert a YAML configuration file to json, and vice-versa. The module supports both the functions of read and write either a single YAML document or a YAML stream.
+
```
Comment (shafreenAnfar) [inline docs/spec/spec.md:11]: Can't we directly bind it to a record?

**Comment by shafreenAnfar** [comment]
Did we use any tools such as grammerly to validate the text?

**Comment by madhukaw** [comment]
> Did we use any tools such as grammerly to validate the text?

Validated through Grammarly and updated in [352e0f5](https://github.com/ballerina-platform/module-ballerina-yaml/pull/3/commits/352e0f548f6caf684401a28f15afaef0ba7a94ab)

File: docs/spec/spec.md (hunk 2 above)
Comment (MadhukaHarith92) [reply docs/spec/spec.md:11]: Currently, the APIs are bound to JSON. Will create and issue to bind it directly to a record.

Comment (madhukaw) [reply docs/spec/spec.md:11]: Currently, APIs are bound to JSON. Will create an issue to directly bind it to a record.

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**